packages/core/contracts/financial-templates/common/financial-product-libraries/KpiOptionsFinancialProductLibrary.sol

```diff
@@ -11,6 +11,42 @@ import "../../../common/implementation/Lockable.sol";
  * Post-expiry, the collateral requirement is left as 1 and the price is left unchanged.
  */
 contract KpiOptionsFinancialProductLibrary is FinancialProductLibrary, Lockable {
+    using FixedPoint for FixedPoint.Unsigned;
+
+    mapping(address => FixedPoint.Unsigned) financialProductTransformedPrices;
```

Comment:
think you should explicitly set this as a `public` var

Reply:
agreed